src/scenes/home/signup/signup.js

```diff
@@ -122,8 +122,8 @@ class SignUp extends Component {
         <Form className={styles.signupForm}>
           <span>
             Are you ready to deploy your future? Our community supports active duty and veterans
-            transitioning into the tech industry, but we welcome spouse, family members,
-            tech professionals, volunteers and sponsors. Join Operation Code today
+            transitioning into the tech industry, but spouse, family members,
```

Comment:
@rahulravindran2709 Sorry, this is my fault, I made a typo on my comment yesterday - can you please change 'spouse' to 'spouses' here? Once that's done this should be good to go.